bdsx/bds/implements.ts

```diff
@@ -1153,6 +1153,14 @@ Block.prototype.isUnbreakable = procHacker.js('?isUnbreakable@Block@@QEBA_NXZ',
 Block.prototype.buildDescriptionId = procHacker.js('?buildDescriptionId@Block@@QEBA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@XZ', CxxString, {this: Block, structureReturn: true});
 Block.prototype.isCropBlock = procHacker.js('?isCropBlock@Block@@QEBA_NXZ', bool_t, {this: Block});
 Block.prototype.popResource = procHacker.js('?popResource@Block@@QEBAPEAVItemActor@@AEAVBlockSource@@AEBVBlockPos@@AEBVItemInstance@@@Z', ItemActor, {this: Block}, BlockSource, BlockPos, ItemStack);
+Block.prototype.canHurtAndBreakItem = procHacker.js('?canHurtAndBreakItem@Block@@QEBA_NXZ', bool_t, {this: Block});
+Block.prototype.getThickness = procHacker.js('?getThickness@Block@@QEBAMXZ', float32_t, {this: Block});
+Block.prototype.hasComparatorSignal = procHacker.js('?hasComparatorSignal@Block@@QEBA_NXZ', bool_t, {this: Block});
+Block.prototype.getTranslucency = procHacker.js('?getTranslucency@Block@@QEBAMXZ', float32_t, {this: Block});
+Block.prototype.getExplosionResistance = procHacker.js('?getExplosionResistance@Block@@QEBAMPEAVActor@@@Z', int16_t, {this: Block}, Actor);
```

Comment:
I think it would be better if the first parameter was optional.

Reply:
How should I write?


> I think it would be better if the first parameter was optional.

Reply:
> How should I write?
> 
> > I think it would be better if the first parameter was optional.

Please check `Mob.prototype.konckback`.
if you have a hard to understanding, that's ok. I will do that.

Reply:
> > How should I write?
> > > I think it would be better if the first parameter was optional.
> 
> Please check `Mob.prototype.konckback`. if you have a hard to understanding, that's ok. I will do that.

I added it.

Reply:
> > > How should I write?
> > > > I think it would be better if the first parameter was optional.
> > 
> > 
> > Please check `Mob.prototype.konckback`. if you have a hard to understanding, that's ok. I will do that.
> 
> I added it.

nice.

but there is one more thing to fix.
when it is a default parameter, it can be undefined and incompatible with `NativeClass`es, like `Actor`.
so you have to do like `Spawner.prototype.spawnMob`